lab-keiran/test/test.js

```diff
@@ -0,0 +1,8 @@
+const cowsay = require('../cowsay');
+const expect = require('chai').expect;
+
+describe('the cowsay function', function() {
+  it('should return what the cow says', function() {
+    expect(cowsay(text)).to.eql.('the cow says' + text);
```

Comment:
syntax error: remove period after eql :  `eql.(`
`eql` is a function your calling with the `()`, `.` are for accessing properties on objects

the word chaining in chai with (`.to.be` etc) is syntactic sugar and isn't necessary, personally it kinds weirds me out and i leave it out:  `cowsay(text)).eql('the cow says '+ text)`